src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -489,6 +489,8 @@ public abstract partial class JsonConverter<T> : System.Text.Json.Serialization.
     {
         protected internal JsonConverter() { }
         public override bool CanConvert(System.Type typeToConvert) { throw null; }
+        public virtual bool HandleNull { get { throw null; } }
+        [return: System.Diagnostics.CodeAnalysis.MaybeNull]
```

Comment:
Doesn't the same feedback from https://github.com/dotnet/runtime/pull/35826#discussion_r421023122 appies to this  and to all other generic types annotated?

Reply:
These annotate the return values, not input parameters. These annotations correctly note that `null` may be returned even though the type may be non-nullable (depending on the JSON payload).

Reply:
Ah I see what you mean. Even though `T` was a non-nullable reference type, we may return `T?`.

Reply:
> Even though T was a non-nullable reference type, we may return T?.

Even if T is a non-nullable reference type, we may return `null`.